app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
```

Comment:
- [ ] It would be nice if you fix your associations here as well because the comments table has a foreign key `user_id` not the `author_id` so it would be nice if you make changes here accordingly :+1: